app/controllers/users_controller.rb

```diff
@@ -3,7 +3,7 @@
 class UsersController < ApplicationController
   def index
     authorize User
-    @users = policy_scope User.all
+    @users = policy_scope User.order('LOWER(first_name)')
```

Comment:
`schema.rb` alapján nem csak `first_name` -je, hanem `last_name` -je is van a `User` -eknek. Szerintem érdemes lenne ezt is beletenni az `order` clause-ba.